fah_xchem/analysis/website/templates/retrospective_relative_free_energies/index.html

```diff
@@ -0,0 +1,139 @@
+{% extends "base.html" %}
+{% import "postera.html" as postera %}
+{% set active_page = "retrospective_microstate_transformations" %}
+{% block content %}
+<h3>Retrospective Transformations <i type="button" class="fa fa-info-circle" data-html="true" data-container="body" data-toggle="popover" data-trigger="hover" data-placement="bottom" data-title="<b>Retrospective transformations</b>" data-content="This page contains a retrospective analysis of the accuracy of individual transformations (without DiffNet MLE estimates) compared with experimental data."></i></th></h3>
```

Comment:
Wait, this has the same header as `retrospective_microstate_transformations`, but I don't see this template in use on any of our sprints. Is this intentional?

Reply:
I had moved the original Retrospective Transformations to Retrospective Microstate Transformations and Retrospective Compound Transformations. There may still be errant lingering references to the old defunct name.

Reply:
Hmmm, but are we using this `retrospective_relative_free_energies` page at all? We don't have a tab for it showing up in [Sprint 11](https://fah-public-data-covid19-moonshot-sprints.s3.us-east-2.amazonaws.com/dashboards/sprint-11/sprint-11-2021-12-26-P1800_0A-dimer-neutral-restrained/transformations/index-201-300.html), for example.